miso-web/src/main/webapp/WEB-INF/pages/editProject.jsp

```diff
@@ -182,7 +182,7 @@
   jQuery(document).ready(function () {
     var projectDto = ${projectDto};
     var config = ${formConfig};
-    config["createProject"] = true;
+    config["projectId"] = projectDto.id;
```

Comment:
This is not useful to the form, and the rest of the config is not relevant to the list. Instead of adding this to the form config, give the list its own config. Can be inline below: `Project.setListConfig({projectId: projectDto.id})`